packages/syft/src/syft/node/routes.py

```diff
@@ -189,4 +189,8 @@ def register(request: Request, data: bytes = Depends(get_body)) -> Any:
         else:
             return handle_register(data, worker)
 
+    @router.get("/ping", name="ping", status_code=200)
```

Comment:
@madhavajay

Reply:
We have one called `healthcheck` in main.py maybe we can use the same one in both places. The other one is only available on `/` for containers which arent routing to the full URL. I think we might need to add this so we can deploy our containers to `CloudRun` anyway, so it would be helpful.

So maybe?
`/healthcheck`